services/credential-issuance-server/src/routes.ts

```diff
@@ -14,6 +14,7 @@ router.get(config.path.shorten, getFullUrl);
 router.post(config.path.createShorten, createShortenUrl);
 router.get(config.path.keriOobi, keriOobiApi);
 router.post(config.path.issueAcdcCredential, issueAcdcCredential);
+router.post(config.path.issueLeCredential, issueLeChainedCredential);
```

Comment:
I think we can just keep the 1 endpoint. But if we try to issue a LE credential, we can issue a QVI credential and LE together in one go?